### ISSUE TITLE
Monitoring WIFI clients stats ?

### ISSUE THREAD

**Issue by luponata** [patricegautier/unifiZabbix#111]
Looking on the "Client devices" tab, information on the WIFI connection of each device is displayed, example:

![zabbix2](https://github.com/patricegautier/unifiZabbix/assets/10811598/efdf21f8-56e9-41dd-8772-f36582f2d8b3)

Would it be possible to choose which client to monitor?, I'm not saying all the devices via a discovery rule because it could be "expensive", but in my case I have a separate access point where I only connect one security camera, and I would like to monitor its connection quality (Signal, TX/RX rate for example)

**Comment by patricegautier** [comment]
The AP templates currently don't support singling out a client like that..   It's theoretically possible to extend them to do this I think (the info is provided by mca-dump on the AP) but that's a fair amount of work. Is the security camera a Unifi protect camera per chance?

**Comment by luponata** [comment]
Yes, you guessed it!, it's a somfy outdoor camera, they just changed it under warranty, because the WIFI performance was poor despite there being no problems with the radio spectrum, I discovered that the new one also had problems and I wanted to keep track of all these variations on signal strength and TX/RX rate. I'll try to take a look at the mca dump out of curiosity
